generator/src/pkg/model/input.go

```diff
@@ -17,21 +17,42 @@ limitations under the License.
 package model
 
 type CalledService struct {
-	Service             string  `json:"service"`
-	Port                string  `json:"port"`
-	Endpoint            string  `json:"endpoint"`
-	Protocol            string  `json:"protocol"`
-	TrafficForwardRatio float32 `json:"traffic_forward_ratio"`
+	Service             string `json:"service"`
+	Port                string `json:"port"`
+	Endpoint            string `json:"endpoint"`
+	Protocol            string `json:"protocol"`
+	TrafficForwardRatio int    `json:"traffic_forward_ratio"`
+	RequestPayloadSize  int    `json:"request_payload_size"`
+}
+
+type CpuComplexity struct {
```

Comment:
We need to define optional values for the required values here, so that if the user skip this values, we could generate reasonable yaml files.